utils_test.go

```diff
@@ -0,0 +1,36 @@
+package testcontainers_test
```

Comment:
Is there a reason for this file to be here? I cannot see usages of the functions on it in the current changeset

Reply:
These are utilities that are currently in the `docker_test` file and are used in the modified tests. Since the `docker_test` file isn't moved yet, I've temporarily created this utils file, it should go away when every thing is migrated 🙂